src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -140,7 +140,6 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                         if (isset($valueMap[$value])) {
                             $knownValues[$child->getName()] = $value;
                             unset($unknownValues[$value]);
-                            continue;
```

Comment:
Let's keep the continue and remove the else instead?

Reply:
Hesitated on that exact one! It's updated